composer.json

```
@@ -4,7 +4,7 @@
     "description": "A Flow package that allows for protecting persistent resources from unauthorized access",
     "license": "MIT",
     "require": {
-        "neos/flow": "^4.1 || ^5.0"
+        "neos/flow": "^4.1 || ^5.0 || ^6.0"
```

This version won't work for Flow versions below 6.0 =>
```suggestion
        "neos/flow": "^6.0"
```

---

Thank you for your feedback. I adjusted the composer.json according to your suggestion